api/types/trust.go

```diff
@@ -40,14 +40,17 @@ const (
 	// certificates but rather is an authority that signs tokens, however it behaves
 	// much like a CA in terms of rotation and storage.
 	JWTSigner CertAuthType = "jwt"
+	// SAMLIDPCA identifies the certificate authority that will be used by the
+	// SAML identity provider.
+	SAMLIDPCA CertAuthType = "saml-idp"
```

Comment:
I think that snake_case is more common than kebab-case for things like this, so I vote for
```suggestion
	SAMLIDPCA CertAuthType = "saml_idp"
```

Reply:
Then again, do we expect to need a separate signer for other types of IdP? Can we just make this `idp`?

Reply:
I would argue that we should have separate signers for other types of `idp` just in case -- say the key for an `oidc_idp` was compromised but not the `saml` one, for example, you wouldn't have to update the IdP metadata for all SAML service providers.